src/routes/landing/components/FilterableCitySelector.js

```diff
@@ -8,34 +9,46 @@ import CityModel from 'modules/endpoint/models/CityModel'
 
 import style from './FilterableCitySelector.css'
 import { translate } from 'react-i18next'
+import type { I18nTranslateType } from 'flowTypes'
 
-export class FilterableCitySelector extends React.Component {
-  static propTypes = {
-    cities: PropTypes.arrayOf(PropTypes.instanceOf(CityModel)).isRequired,
-    language: PropTypes.string.isRequired,
-    t: PropTypes.func.isRequired
-  }
+type PropsType = {
+  cities: Array<CityModel>,
+  language: string,
+  t: I18nTranslateType
+}
+
+type StateType = {
+  filterText: string
+}
 
-  constructor (props) {
+export class FilterableCitySelector extends React.Component<
+  PropsType,
+  StateType
+> {
```

Comment:
maybe we need to adjust our maximum line width, so we don't need too much of these line breaks.

Reply:
Yes, that looks odd. If we change a eslint rule I won't do it in this task.

But I'll open an issue for this.